index.js

```diff
@@ -81,7 +81,7 @@ function _parseInput (input, opts, cb) {
 
   var commonPrefix = null
   input.forEach(function (item, i) {
-    if (typeof item === 'string') return
+    if (typeof item === 'string' || (Buffer.isBuffer(item) && !item.name)) return
```

Comment:
Yeah, every file in a torrent needs a name. There's a file list in the .torrent file. If the user passes in a buffer without a name, we need to either:

1. Throw an exception so they can add a name
2. Add a name for them